package.json

```diff
@@ -1,22 +1,20 @@
 {
-  "name": "signin",
-  "version": "0.0.0",
+  "name": "checkin",
+  "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "http-server ./app -p 3000 -c-1",
+    "test": "karma start ./test/karma.conf.js"
   },
   "dependencies": {
-    "body-parser": "~1.6.6",
-    "cookie-parser": "~1.3.2",
-    "csurf": "^1.7.0",
-    "debug": "~1.0.4",
-    "ejs": "^2.2.4",
-    "express": "~4.8.6",
-    "express-simple-router": "0.0.3",
-    "jade": "~1.5.0",
-    "morgan": "~1.2.3",
-    "request": "^2.53.0",
-    "request-promise": "^0.4.0",
-    "serve-favicon": "~2.0.1"
+    "coveralls": "^2.11.2"
```

Comment:
Nope! I got this from the Intranet repo tbh - https://github.com/TechAtNYU/intranet/blob/develop/package.json